<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>367</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>366</width>
    <height>367</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>366</width>
    <height>367</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img/res.qrc">
    <normaloff>:/icon/calculator.png</normaloff>:/icon/calculator.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>0</y>
      <width>362</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color:  #4B0082;
	color: yellow;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus_minus">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_procent">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: green;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: blue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: blue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: green;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: blue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>183</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: blue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>244</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: blue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>183</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: blue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>183</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: green;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>244</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: blue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>244</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: green;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>183</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: blue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>244</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: blue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: green;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>305</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: blue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqr">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>sqr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_one_div">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>183</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m_save">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>244</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>MS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m_res">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>MR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m_plus">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>183</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>M+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m_clear">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>MC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m_minus">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>244</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>M-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pi">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>Pi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_e">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: white;
}

 QPushButton:pressed {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
	color: black;
 }</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img/res.qrc"/>
 </resources>
 <connections/>
</ui>
